src/VisualStudio/Core/Def/ExternalAccess/UnitTesting/Api/UnitTestingRemoteHostOptionsAccessor.cs

```diff
@@ -2,17 +2,16 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Linq;
 using Microsoft.CodeAnalysis.Options;
 using Microsoft.VisualStudio.LanguageServices.Remote;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
 {
     internal static class UnitTestingRemoteHostOptionsAccessor
```

Comment:
>UnitTestingRemoteHostOptionsAccessor [](start = 26, length = 36)

This shouldn't be used anymore.

Reply:
➡️ This is part of the API of the restricted IVT, so we have to maintain functional compatibility to the degree possible. If the type is no longer used by partner code, it can be removed as a separate change.